### ISSUE TITLE
Unable to Switch Tabs After Full-Screen Video

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#13264]
## Description 
A user on the Brave Community site has [shared an issue](https://community.brave.com/t/tabs-not-working/188880) impacting their _Samsung Galaxy Tab S7+_. After opening a few tabs, navigating to YouTube, and briefly playing a video in full-screen, they are no longer able to switch tabs.

## Steps to reproduce 
   1. Launch Brave and open 4 tabs
   2. Navigate to YouTube in one tab, and play a video in full-screen
   3. Close the full-screen video, and attempt to select another tab

## Actual result 
The user's tap is not registered, and/or the unfocused tab is not focused.

## Expected result
Once the full-screen video is closed, the user should be able to switch tabs easily.

## Issue reproduces how often 
Easily reproducible.

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? Yes
- Can you reproduce this issue with the current Play Store Beta version? N/A 
- Can you reproduce this issue with the current Play Store Nightly version? N/A

## Device details
   - Install type (ARM, x86):
   - Device type (Phone, Tablet, Phablet): Samsung Galaxy Tab S7+
   - Android version: Latest, according to user.
   
## Brave version
v1.17.75

**Comment by demodanza** [comment]
I can confirm this happening, although I'm not sure is related to video, as sometimes it happens in regular browsing, especially in landscape. I have to turn the tablet to portrait and then back to landscape to be able to interact with the tabs again. 

Happens in my tablet
Mipad 4, MIUI 10.3.2
Android 8.1.0
3 GB ram, Octa-core max 2.20 GHz

And in my girlfriend's, a Huawei Media pad T5

**Comment by flexphperia** [comment]
I confirm I have the same problem on iPlay 20 pro tablet. Brave 1.18.75

**Comment by coyoteodduse** [comment]
I too can confirm I have the same problem.
Samsung Galaxy Tab S7 Model SM-T510
Brave version 1.18.75

**Comment by dagonlore** [comment]
This bug can be found on:
Brave version 1.18.75
Brave Beta version 1.19.72

The Brave Nightly version 1.20.54 does not reproduce the above issue. (I am unable to recreate it at this time)

My Galaxy tab S7+ (SM-T970) is running Android 10
I don't think it matters but I have the Wifi only model with 256/8GB
Happy New year everyone 🥳

**Comment by dagonlore** [comment]
Brave version 1.18.77 has fixed the above issue.
Thank you to everyone who put in countless hours troubleshooting this bug.

**Comment by bsclifton** [comment]
Awesome, thanks for confirming @dagonlore 😄  I'll put this into the 1.20 milestone so that we can officially verify this is fixed

**Comment by dagonlore** [comment]
Thanks for the quick response.
I'm going to make another video of another issue here soon and I'll start a new thread.
🙃

R/S

Ben
________________________________

**Comment by coyoteodduse** [comment]
installed 1.18.77    NOT FIXED

**Comment by coyoteodduse** [comment]
installed 1.18.77    NOT FIXED

**Comment by coyoteodduse** [comment]
installed 1.18.77    NOT FIXED

**Comment by dagonlore** [comment]
Maybe my tab S7+ was the only device to get the fix, idk... That's very strange.  I'll look into this to see if I can change some of my recent device settings to try to bring this bug back.
Anyway, good luck fixing my newest submission.



R/S

Ben



-------- Original message --------

**Comment by dagonlore** [comment]
First I would like to apologise.
I preformed poor troubleshooting steps.
I had forgotten that I changed a few system settings right before I updated the app.
I made a video of the two system changes that fixed this issue for me.
Here is that video.
https://youtu.be/2prcksCJtYg
I am not sure that this qualifies as a fix as some devices may not have these system settings or even want to make the changes.
I am sorry again, I will do better in the future.

**Comment by flexphperia** [comment]
I confirm nothing changed, problem still persist in 1.18.77 on iPlay 20 pro tablet and other tablets.

**Comment by dimon222** [comment]
There's 1.18.78 on Play Store. Worth to check.

**Comment by coyoteodduse** [comment]
Thanks but not fixed.

**Comment by bsclifton** [comment]
Re-opened after above comments

**Comment by demodanza** [comment]
I can confirm it's still happening on my end, using the latest version.

Mipad 4, MIUI 10.3.2
Android 8.1.0
3 GB ram, Octa-core max 2.20 GHz

And Huawei Media pad T5

**Comment by dagonlore** [comment]
This is the last time I am going to post about this topic.
My Galaxy Tab S7+ just received the new Android 11 update, the UI 3.1.
I updated the Brave app to Brave 1.19.88 
I also changed my tablets system settings back to the "out of box" settings.
I am unable to recreate this issue on my tablet.
Even though the issue did not effect me since changing my system setting from when I said the issue no longer effect me I decided to see how things worked with the new Android 11 update.
It does not matter what I do on my end, I am unable to recreate this issue.
I just hope that everyone else that still has this issue will get a fix soon.
